### ISSUE TITLE
LEGO Dimensions

### ISSUE THREAD

**Issue by darkuser89** [SamoZ256/cemu-metal-game-compatibility#80]
## Description

`Boot`

## Screenshots

Only Blackscreen 
[log.txt](https://github.com/user-attachments/files/18143311/log.txt)

## Cemu Metal version

`testing1.0.9`

## Mac

M3 Pro // 15.2 

## Similar reports

- [X] I have checked that the game hasn’t been reported on the `Issues` page yet (please tick the checkbox by writing `X` inside `[ ]`)

**Comment by SamoZ256** [comment]
Is there black screen all the time? Or does at least something render? I think I remember Hakase testing this game in the Discord and it was at least partially working

**Comment by darkuser89** [comment]
no, at the beginning you can see the pictures with all the logos and such.
<img width="1159" alt="Bildschirmfoto 2024-12-16 um 06 35 47" src="https://github.com/user-attachments/assets/46885029-5fd0-466f-8d27-ff20d5508c5a" />

 but after the loading screen. The picture remains black and you only hear the sound

**Comment by shinra-electric** [comment]
> Is there black screen all the time? Or does at least something render? I think I remember Hakase testing this game in the Discord and it was at least partially working

Nope, wasn't me. I don't have any of the Lego games...

**Comment by Sonicloop-dev** [comment]
Hi, I know this issue is still being worked on and I would love to play the whole Lego dimensions. But if you remove the update the game is playable on version 0. I believe something changed in the rendering of the game in one of the updates. But on version 0 the base story is playable and a few hub worlds like the lego movie and the DC universe one. I would love for the rest of the game to be playable tho.
